### PR TITLE
change to template

### PR DESCRIPTION
**Pull request by daroga0002**
This is changing copy operation to template module. This will give you more customization options for that.

### CODE REVIEW

File: main.yml [hunk 1]
```diff
@@ -53,8 +53,8 @@
       - restart_ssh
 
   - name: Message na starcie
-    ansible.builtin.copy:
-      src: files/issue
+    ansible.builtin.template:
```
Comment (marekmleple) [inline main.yml:56]: templates do doopy

File: main.yml [hunk 2]
```diff
@@ -53,8 +53,8 @@
...
+      src: templates/issue.j2
```
Comment (marekmleple) [inline main.yml:57]: ok

**Review by marekmleple — APPROVED** [review]

File: main.yml (hunk 1 above)
Comment (daroga0002) [reply main.yml:56]: ok, this looks good

File: main.yml (hunk 2 above)
Comment (daroga0002) [reply main.yml:57]: ok

**Pull request merged**